repo/packages/E/etcd/0/marathon.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "{{etcd.framework-name}}",
+    "container": {
+        "docker": {
+            "image": "{{etcd.docker-image}}",
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{etcd.cpus}},
+    "mem": {{etcd.mem}},
+    "instances": 1,
+    "ports": [0, 0, 0],
+    "env": {
+      "FRAMEWORK_NAME":"{{etcd.framework-name}}",
+      "CLUSTER_SIZE":"{{etcd.cluster-size}}",
+      "MESOS_MASTER":"{{mesos.master}}",
+      "ZK_PERSIST":"{{etcd.zk}}",
+      "VERBOSITY":"{{etcd.logging-verbosity}}",
+      "AUTO_RESEED":"{{etcd.auto-reseed}}",
+      "RESEED_TIMEOUT":"{{etcd.reseed-timeout}}",
+      "DISK_LIMIT":"{{etcd.disk-limit}}",
+      "CPU_LIMIT":"{{etcd.cpu-limit}}",
+      "MEM_LIMIT":"{{etcd.mem-limit}}",
+      "WEBURI": "http://{{etcd.framework-name}}{{etcd.dns-suffix}}:{{etcd.admin-port}}/stats"
+    },
+    "healthChecks": [{
+      "protocol": "HTTP",
+      "path": "/healthz",
```

Comment:
The idea is to not restart at all, but set the grace period or allowed failure count to a high value. Marathon really needs health checks with restart behavior and some without.